eclair-core/src/test/scala/fr/acinq/eclair/TestConstants.scala

```diff
@@ -143,8 +158,12 @@ object TestConstants {
       pingInterval = 30 seconds,
       pingTimeout = 10 seconds,
       pingDisconnect = true,
-      maxFeerateMismatch = 1.0,
-      updateFeeMinDiffRatio = 0.1,
+      onChainFeeConf = OnChainFeeConf(
```

Comment:
Not a big issue but I don't like the name, I would call it either `OnChainFeeConfiguration` or  `OnChainFeeSettings`

Reply:
I tried to be consistent with `RouterConf` but i reckon is not a great name (especially because it recalls "configuration" while it contains an estimator too).

Reply:
Well this is taken from `reference.CONF` so it's not that bad.